misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordConverterService.java

```diff
@@ -45,13 +45,15 @@
     @JsonRequest
     public CompletableFuture<JsonToRecordResponse> convert(JsonToRecordRequest request) {
         return CompletableFuture.supplyAsync(() -> {
-            JsonToRecordResponse response = new JsonToRecordResponse();
+            JsonToRecordResponse response;
             try {
                 String jsonString = request.getJsonString();
-                String codeBlock = JsonToRecordConverter.convert(jsonString);
-                response.setCodeBlock(codeBlock);
+                String recordName = request.getRecordName();
+                boolean isRecordTypeDesc = request.getIsRecordTypeDesc();
+                boolean isClosed = request.getIsClosed();
+                response = JsonToRecordConverter.convert(jsonString, recordName, isRecordTypeDesc, isClosed);
             } catch (IOException | JsonToRecordConverterException | FormatterException e) {
-                response.setCodeBlock("");
+                response = new JsonToRecordResponse();
```

Comment:
Can we also set an empty string as the codeblock here; as the vscode plugin checks for an empty string in case of an error.

Reply:
This is fixed

Reply:
Resolved in https://github.com/ballerina-platform/ballerina-lang/pull/33663/commits/bcf590f93a9af08bae304c79d752b7a8a45e29a0